client/blocks/importer/wordpress/import-everything/index.tsx

```diff
@@ -1,381 +0,0 @@
-import { PLAN_MIGRATION_TRIAL_MONTHLY } from '@automattic/calypso-products';
-import {
-	MigrationStatus,
-	type MigrationStatusError,
-	type SiteDetails,
-} from '@automattic/data-stores';
-import { Hooray, SubTitle, Title } from '@automattic/onboarding';
-import { createElement, createInterpolateElement } from '@wordpress/element';
-import { localize } from 'i18n-calypso';
-import { get } from 'lodash';
-import { connect } from 'react-redux';
-import ImportUsers from 'calypso/blocks/importer/wordpress/import-everything/import-users';
-import PreMigrationScreen from 'calypso/blocks/importer/wordpress/import-everything/pre-migration';
-import AsyncLoad from 'calypso/components/async-load';
-import { LoadingEllipsis } from 'calypso/components/loading-ellipsis';
-import { EVERY_FIVE_SECONDS, Interval } from 'calypso/lib/interval';
-import { SectionMigrate } from 'calypso/my-sites/migrate/section-migrate';
-import { isMigrationTrialSite } from 'calypso/sites-dashboard/utils';
-import { recordTracksEvent } from 'calypso/state/analytics/actions';
-import getCurrentQueryArguments from 'calypso/state/selectors/get-current-query-arguments';
-import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
-import { receiveSite, requestSite, updateSiteMigrationMeta } from 'calypso/state/sites/actions';
-import { getSite, getSiteAdminUrl, isJetpackSite } from 'calypso/state/sites/selectors';
-import { IAppState } from 'calypso/state/types';
-import DomainInfo from '../../components/domain-info';
-import DoneButton from '../../components/done-button';
-import NotAuthorized from '../../components/not-authorized';
-import { isTargetSitePlanCompatible } from '../../util';
-import { WPImportOption, type MigrationState } from '../types';
-import { clearMigrateSource, retrieveMigrateSource } from '../utils';
```

Comment:
`clearMigrateSource` doesn't seem to be necessary anymore

Reply:
Yup, although I wrote some considerations about it in [this comment](https://github.com/Automattic/wp-calypso/pull/102282#discussion_r2022718358)

Reply:
Yeah. Not a huge win anyway.

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/102282/commits/ee5438cad8d09b60d0d85a9f4baca0f372448826 as part of a bigger cleanup